backend/app/main.py

```diff
@@ -18,6 +21,20 @@ async def running_total(purchase: Purchase):
     return calc_purchase(purchase) if purchase else 0
 
 
+@app.get("/cellphones", response_model=list[CellPhone])
+async def get_cellphones():
+
+    services = [
+        CellPhone(name="Motorola G99", price=800, regularity=Regularity.ONCE),
+        CellPhone(name="iPhone 99", price=6000, regularity=Regularity.ONCE),
+        CellPhone(name="Samsung Galaxy 99", price=1000, regularity=Regularity.ONCE),
+        CellPhone(name="Sony Xperia 99", price=900, regularity=Regularity.ONCE),
+        CellPhone(name="Huawei 99", price=900, regularity=Regularity.ONCE),
```

Comment:
You don't have to set the regularity, ONCE should be the default value

Reply:
Coding at 1:30 in the morning be like